source/clear-linux/get-started/virtual-machine-install/vmw-player-preconf.rst

```diff
@@ -0,0 +1,314 @@
+
+      #. Start the installer by issuing the command below and follow the guided
+         steps.
+
+         .. code-block:: console
+
+            sudo sh ./VMware-Player-[version number].x86_64.bundle
+
+   * On Windows:
+
+      #. Start the installer.
+      #. Follow the setup wizard.
+
+For additional help, see the `VMware Workstation Player guide`_.
+
+Clear Linux image types
+***********************
+
+.. include:: ../../guides/maintenance/image-types.rst
+   :Start-after: image-types-content:
+
+Download the latest Clear Linux VMware image
+********************************************
+
+Get the latest |CL| VMware image from the `image`_ repository.
+Look for :file:`clear-[version number]-vmware.vmdk.xz`.
+
+.. include:: ../../guides/maintenance/download-verify-uncompress-windows.rst
+   :Start-after: verify-windows:
+
+We also provide instructions for other operating systems:
+
+* :ref:`download-verify-uncompress-linux`
+* :ref:`download-verify-uncompress-mac`
+
+Create and configure a new VM
+*****************************
+
+#. Start the `VMware Workstation Player` app.
+#. On the home screen, click :guilabel:`Create a New Virtual Machine`. See
+   figure 1.
+
+   .. figure:: figures/vmw-player-preconf/vmw-player-preconf-01.png
+      :scale: 100%
+      :alt: VMware Workstation 14 Player - Create a new virtual machine
+
+      Figure 1: VMware Workstation 14 Player - Create a new virtual machine
+
+#. On the :guilabel:`Welcome to the New Virtual Machine Wizard` screen, select
+   the :guilabel:`I will install the operating system later` option.
+   See figure 2.
+
+   .. figure:: figures/vmw-player-preconf/vmw-player-preconf-02.png
+      :scale: 100%
+      :alt: VMware Workstation 14 Player - Select install operating system
+
+      Figure 2: VMware Workstation 14 Player - Select install operating system
+      later.
+
+#. Click the :guilabel:`Next` button.
+
+#. On the :guilabel:`Select a Guest Operating System` screen, set the
+   :guilabel:`Guest operating system` setting to :guilabel:`Linux`.
+   See figure 3.
+
+   .. figure:: figures/vmw-player-preconf/vmw-player-preconf-03.png
+      :scale: 100%
+      :alt: VMware Workstation 14 Player - Select guest operating system type
+
+      Figure 3: VMware Workstation 14 Player - Select guest operating system
+      type
+
+#. Set :guilabel:`Version` setting to
+   :guilabel:`Other Linux 3.x or later kernel 64-bit`.
+
+#. Click the :guilabel:`Next` button.
+
+#. On the :guilabel:`Name the Virtual Machine` screen, give your new VM a name.
+   See figure 4.
+
+   .. figure:: figures/vmw-player-preconf/vmw-player-preconf-04.png
+      :scale: 100%
+      :alt: VMware Workstation 14 Player - Name virtual machine
+
+      Figure 4: VMware Workstation 14 Player - Name virtual machine
+
+#. Click the :guilabel:`Next` button.
+
+#. On the :guilabel:`Specify Disk Capacity` screen, click the :guilabel:`Next`
+   button. The disk settings do not matter at this stage because you will remove
```

Comment:
The wording added is too complex and references future steps without specificity which causes confusion in readers. Both things were avoided in the previous version. I suggest the following alternative:
Keep the default disk settings unchanged. When we attach the pre-configured |CL| VMware image, VMWare removes the default virtual disk with the pre-configured disk.